public/javascripts/purchases/purchases.js

```diff
@@ -0,0 +1,101 @@
+var purchaseSheetKey = '1HFk8TaGWq5j9mouKGJYQeD0Xb9KxvWiXvtfs8_091A8';
```

Comment:
My bad, fortunately this is just the key that links to the test sheet I set up, not any user IDs or auth keys